### ISSUE TITLE
[css-anchor-position] position-try: inset-area() does not reflect CSSWG resolution

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#10320]
The [minutes of the CSSWG resolution](https://github.com/w3c/csswg-drafts/issues/9196#issuecomment-1942197882) to add `position-try` include a resolution to add the `inset-area` syntax into the [position-try proposal](https://github.com/w3c/csswg-drafts/issues/9196#issuecomment-1912788896).

However, what got edited in was not what was discussed (adding `<'inset-area'>` as an option in the try list), but a [new `inset-area()` function](https://drafts.csswg.org/css-anchor-position-1/#position-try-options). This is inconsistent both with how we import syntaxes across properties in general (we don't wrap them in a function) and with the WG resolution as most reasonably interpreted (given the logged discussion, which explicitly noted the syntax proposal prior to the resolution).

**Comment by tabatkins** [comment]
I don't read the minutes as being particularly specific about the syntax. Notably, the syntax suggestion was introduced by a comment in the IRC discussion, not discussed in the thread at all, and unless I'm missing a comment, there was no discussion from anyone else about the syntax specifically during the meeting. We've similarly interpreted this sort of resolution somewhat loosely wrt precise syntax in the past, once the editors dig in and decide it would be better written slightly differently, so long as the addition agrees with the spirit of the resolution.

That's what happened here - once I got to writing it in, I found the inset-area syntax to not be particularly clear to include directly, and thought the wrapping function made it read much better. I'm also slightly concerned about future extensibility here - if we add anything else that we want to make inline-able, we'll run into similar issues.

We don't anticipate any issues in changing this to be directly embedding the syntax, I just think we shouldn't do it because I think the current syntax reads better and seems more safely extensible.
